swift/Sources/FlatBuffers/Mutable.swift

```diff
@@ -25,7 +25,7 @@ public protocol Mutable {
   /// makes Flatbuffer accessed within the Protocol
   var bb: ByteBuffer { get }
   /// makes position of the ``Table``/``struct`` accessed within the Protocol
-  var postion: Int32 { get }
+  var position: Int32 { get }
```

Comment:
this is the main reason for this PR -- I've left it as a distinct commit from a general `position` fix to make it more easy to review.